docs/api-reference/SPORTS.md

```diff
@@ -26,14 +26,35 @@ Returns a `string` containing the current record of the Men's basketball team.
 ### **get_next_mens_basketball_game()**
 
 #### **Returns**:
-Returns a dictionary containing the full name of the event, the short name of the event, the type of game (regular season, playoffs, preseason, etc.), the week in which it occurs, and the location.
+Returns a `dictionary` containing the time and date of the game in UTC, the opponent's ESPN API ID, school name and full team name, whether Pitt is the home or away team, and information on its location. It also returns a 'Status' attribute, which identifies if the game has completed but data in the API has not updated yet.
 #### **Example**:
 
 ###### **Sample Usage**:
 ```python
 >>> next_game_details = get_next_mens_basketball_game()
 >>> print(next_game_details)
-{'full_name': 'Pittsburgh Panthers at Syracuse Orange', 'short_name': 'PITT @SYR', 'season_name': 'Regular Season', 'week': 'Week 19', 'court': 'Spectrum Center'}
+{
+    'Timestamp': '2022-03-08T19:00Z',
```

Comment:
can we maintain the standard of underscore separated key names?

Reply:
Resolved in [afdce7b](https://github.com/pittcsc/PittAPI/pull/137/commits/afdce7b22c50d0c6f18af707f379a39adcacdd4a)